autogpt_platform/frontend/src/components/agptui/FeaturedAgentCard.tsx

```diff
@@ -47,17 +51,17 @@ export const FeaturedAgentCard: React.FC<FeaturedStoreCardProps> = ({
             />
           </div>
           <div
-            className={`absolute inset-0 overflow-y-auto transition-opacity duration-200 ${
+            className={`absolute inset-0 overflow-y-auto p-4 transition-opacity duration-200 ${
               isHovered ? "opacity-100" : "opacity-0"
-            } rounded-xl dark:bg-neutral-700`}
+            }`}
           >
-            <p className="text-base text-neutral-800 dark:text-neutral-200">
+            <CardDescription className="line-clamp-[11] text-sm leading-relaxed text-muted-foreground">
```

Comment:
We will want to keep mobile in mind when building out specialized components like this. 
It would be good to add a something similar to this:

```suggestion
              <CardDescription className="line-clamp-[6] sm:line-clamp-[8] text-xs sm:text-sm">
                {agent.description}
              </CardDescription>
```